--- catcodec-1.0.1/Makefile~	2010-08-25 18:39:16.000000000 +0200
+++ catcodec-1.0.1/Makefile	2010-09-02 14:58:19.162337310 +0200
@@ -39,7 +39,7 @@
 
 $(CATCODEC): $(OBJS:%=objs/%)
 	@echo '[LINK] $@'
-	$(Q)$(CXX) -o $@ $(CXXFLAGS) $(LDFLAGS) $^
+	$(Q)$(CXX) $(LDFLAGS) -o $@ $(CXXFLAGS) $(LDFLAGS) $^
 
 VERSION := $(shell ./findversion.sh | cut -f 1 -d'	')
 RES := $(shell if [ "`cat version.cache 2>/dev/null`" != "$(VERSION)" ]; then echo "$(VERSION)" > version.cache; fi )
