--- catcodec-1.0.0/Makefile~	2009-11-05 23:33:48.000000000 +0100
+++ catcodec-1.0.0/Makefile	2010-04-23 16:10:15.281761128 +0200
@@ -19,7 +19,7 @@
 OBJS = catcodec.o io.o sample.o rev.o
 OS = unknown
 
-CFLAGS += -Wall -Wcast-qual -Wwrite-strings -Wno-multichar
+CFLAGS += -Wall -Wcast-qual -Wwrite-strings -Wno-multichar $(OPTFLAGS)
 ifdef DEBUG
 	CFLAGS += -g -ggdb
 endif
@@ -30,7 +30,7 @@
 	$(CXX) $(CFLAGS) -c -o $@ $^
 
 $(CATCODEC): $(OBJS)
-	$(CXX) $(CFLAGS) -o $@ $^
+	$(CXX) $(CFLAGS) $(LDFLAGS) -o $@ $^
 
 VERSION := $(shell ./findversion.sh | cut -f 1 -d'	')
 RES := $(shell if [ "`cat version.cache 2>/dev/null`" != "$(VERSION)" ]; then echo "$(VERSION)" > version.cache; fi )
